packages/neuron-wallet/src/controllers/app/index.ts

```diff
@@ -52,11 +62,21 @@ export default class AppController {
     await Promise.all([stopCkbNode(), IndexerService.getInstance().stop()])
   }
 
+  public registerChannels(winId: number, channels: string[]) {
```

Comment:
There's no need to `unregisterChannel`?

Reply:
if the window destroyed, the window id will not be find.

Reply:
In this case, a `WeakMap` could be used to release the `channels` automatically

Reply:
If use `WeakMap`, the WeakMap key type should be `Electron.BrowserWindow`,change like this?
`private windowRegisterChannels = new Map<Electron.BrowserWindow, string[]>()`

Reply:
Is `new Map<Electron.BrowserWindow, string[]>()` a typo? I think it's OK to use `new WeakMap<Electron.BrowserWindow, string[]>()` if window instance is a singleton